packages/analytics/src/rudderstack.ts

```diff
@@ -124,10 +125,15 @@ export class RudderStack {
         }
     }
 
-    identifyEvent = (user_id: string, payload: TEvents['identify']) => {
+    setAccountType(account_type: string) {
+        this.account_type = account_type;
+    }
+
+    identifyEvent = (user_id: string, account_type: string, payload: TEvents['identify']) => {
         if (this.has_initialized) {
             RudderAnalytics.identify(user_id, payload);
             this.has_identified = true;
+            this.account_type = account_type;
```

Comment:
Should `account_type` be included in the `identify` event? In this case you need to put it within the payload itself:
```
RudderAnalytics.identify(user_id, {
   ...payload,
   account_type,
})
```

Reply:
Or better update the type for `TEvents['identify']` to include `account_type: string`

Reply:
I don't think it's the same thing. From what I see from report screenshots which were sent by Denis, `language` doesn't get inside payload, so I explicitly put `account_type` separately to pass it forward. I'm not sure if payload is intended to work like that or it's a bug.

Reply:
This is not required anymore actually, since I didn't intend to pass `account_type` with identify event and only used it to set it to new type here. I refactored it to move it outside of this logic completely.